modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc

```diff
@@ -61,6 +61,28 @@ spec:
     maxPods: 500
 ----
 +
+The rate at which the kubelet talks to the API server depends on qps and burst
```

Comment:
I'm not familiar with the term "qps", but should we standardize on caps or no caps here? Later in this paragraph it's kubelet QPS" and in the YAML it's <qps>.

Reply:
Whoops! Good catch. Thanks!